geopmgo/geopmread.go

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) 2015 - 2024 Intel Corporation
```

Comment:
Probably can be deleted.  It was my first attempt to write some sort of integration test based on copying from the python implementation for geopmread